multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -13,19 +13,39 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
 
     - name: kubelet.service
       enable: true
       command: start
       content: |
+{{ if eq .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=docker.service load-rkt-stage1.service
+        After=docker.service load-rkt-stage1.service
+{{ end }}
         [Service]
         Environment=KUBELET_VERSION={{.K8sVer}}
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
-        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf --mount volume=dns,target=/etc/resolv.conf"
+        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
+        --mount volume=dns,target=/etc/resolv.conf \
+        --volume=mp,kind=host,source=/usr/sbin/modprobe \
+        --mount volume=mp,target=/usr/bin/modprobe \
+        --volume=rkthack,kind=host,source=/etc/rkthack \
+        --mount volume=rkthack,target=/usr/bin/rkt \
+        --volume var-lib-rkt,kind=host,source=/var/lib/rkt \
+        --mount volume=var-lib-rkt,target=/var/lib/rkt \
+        --volume=stage,kind=host,source=/tmp \
+        --mount volume=stage,target=/tmp"
```

Comment:
pod manifests are communicated via tempfiles right now. hardcoding `/tmp/` is only mildly fragile, but should work on coreos just fine